cypress/integration/login.js

```diff
@@ -14,14 +14,32 @@ describe('Sourcebooks login', function() {
         cy.get('[id="loginForm.userId"]').click({force:true});
         cy.get('[aria-label="Demo User"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="User"]').click();
+        cy.get('[aria-label="Team Lead"]').click();
         cy.get('[type="submit"]').click();
 
         cy.url().should('include', '/time-logging');
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User');
-        cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.main-nav').find('li').should('have.length', 2);
+
+        var date1 = new Date;
+        cy.get('.calendar--today').contains(date1.getDate());

```

Comment:
I'd probably also get ".calendar__date" after ".calendar--today", because right now this will consider not only the date but also the number of hours logged :)